parametermanager/api/ParameterManager.Samples.Tests/ParameterManagerFixture.cs

```diff
@@ -0,0 +1,124 @@
+// Copyright(c) 2025 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Iam.V1;
+using Google.Cloud.ParameterManager.V1;
+using Google.Cloud.SecretManager.V1;
+using Google.Protobuf;
+using System.Text;
+
+[CollectionDefinition(nameof(ParameterManagerFixture))]
+public class ParameterManagerFixture : IDisposable, ICollectionFixture<ParameterManagerFixture>
+{
+    public string ProjectId { get; }
+    public const string LocationId = "global";
+
+    public const string Payload = "test123";
+    public const string JsonPayload = "{\"username\": \"test-user\", \"host\": \"localhost\"}";
+    public const string SecretId = "projects/project-id/secrets/secret-id/versions/latest";
+
+    public ParameterName ParameterName { get; }
+    public ParameterName ParameterNameWithFormat { get; }
+
+    public string ParameterId { get; }
+    public Parameter Parameter { get; }
+    public ParameterVersionName ParameterVersionName { get; }
+
+    public Parameter ParameterWithFormat { get; }
+    public ParameterVersionName ParameterVersionNameWithFormat { get; }
+
+    public Parameter ParameterWithSecretReference { get; }
+    public ParameterVersionName ParameterVersionNameWithSecretReference { get; }
```

Comment:
All of these seem to be used in a single test each. If that's the case, they shouldn't be on the fixture, they should be on the specific test that uses them. In the fixture, you should only have data that you share across tests (and that is not modified by tests) and helper methods.

For cleanup, you can have a collection of `_parametersToDelete`, etc. that you iterate over on dispose to delete the parameters. Each test would call the helper methods it needs, for instance  `CreateParameters`, and it's those helper methods the ones that add paremeter names to that collection.